SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -33,11 +52,12 @@ define_repo_path() {
 
 # Function for printing the help page
 print_help() {
-    _log "Usage: makenv --repo/-r REPOSITORY [--repo_type TYPE] [--accpy ACCPY_VERSION] [--help/-h]"
+    _log "Usage: makenv --repo_type TYPE --repo REPOSITORY --builder BUILDER --builder_version VERSION [--help/-h]"
     _log "Options:"
-    _log "  -r, --repo REPOSITORY       Path or http link for a public repository (mandatory)"
-    _log "  --repo_type TYPE            Type of repository (git or eos) (mandatory)"
-    _log "  --accpy VERSION             Version of Acc-Py to be used"
+    _log "  --repo_type TYPE            Type of repository (git or eos)"
+    _log "  --repo REPOSITORY           Path or http link for a public repository"
```

Comment:
Would it make sense to move the `--repo` line before `--repo_type` and not after?

Reply:
I thought that if the `repository` value depends/is defined on/by `repo_type`,  would make more sense to prioritize `repo_type`

Reply:
That would be fine if the second param was called `repo_location`, but being called `repo`... It's a bit like `builder` and `builder_version`.